packages/nep141-erc20/src/bridged-nep141/sendToEthereum/index.js

```diff
@@ -85,6 +86,93 @@ export function checkStatus (transfer) {
   }
 }
 
+// Recover transfer from a withdraw tx hash
+// Track a new transfer at the completedStep = WITHDRAW so that it can be unlocked
```

Comment:
It's not hooked up yet, but most other comments throughout this repo use [typedoc-style notation](http://typedoc.org/guides/doccomments/) so that we can generate great documentation, whether people are relying on inline docs in their editor or viewing it on the web. 

```ts
/**
 * Recover transfer from a withdraw tx hash.
 * Track a new transfer at the completedStep = WITHDRAW so that it can be unlocked
 * @param withdrawTxHash hash of ...
 */
export async function ...
```